go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/containerd/btrfs v1.0.0
 	github.com/containerd/cgroups v1.0.1
 	github.com/containerd/console v1.0.2
+	github.com/containerd/containerd/api v0.0.0
```

Comment:
This currently means "any version will do".

I'm would have to read up on the release procedure, but before a release, we must make sure that this refers to an actual tagged version.

- Do we need to discuss versioning for this? i.e.
   - do we want (or _must_)  `containerd/api version === containerd version`?
   - or will / can the api be versioned separately? (containerd 1.5 using api 1.0)?
- In any case, it likely needs to be added to either a "release checklist", or "release script" to make sure this is don't correctly. e.g. (through script);
    - verify if `git diff <last release> <current code>`:
    - if so: needs a new release tagged
    - update main go.mod with new version
    - tag containerd release

This probably needs changes / additions in [RELEASES.md](https://github.com/containerd/containerd/blob/main/RELEASES.md#public-api-stability)  (public API stability)

Reply:
We should add this as a follow-up issue (but a release blocker). @thaJeztah in the PR message i already linked to your previous comment. https://github.com/containerd/containerd/pull/5716#discussion_r668821396

will add a link to this comment as well.